atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -301,6 +323,8 @@ def render_request(self):
         request = None
         if self.packages:
             request = self.render_packages_request()
+            if self.arches:
+                request['arches'] = self.arches
```

Comment:
Do these two lines belong in `render_packages_request`?

Reply:
Fixed.